temba/api/serializers.py

```diff
@@ -1042,8 +1042,9 @@ class BroadcastCreateSerializer(serializers.Serializer):
     channel = ChannelField(queryset=Channel.objects.filter(pk=-1), required=False)
 
     def __init__(self, *args, **kwargs):
-        self.user = kwargs.pop('user')
-        self.org = self.user.get_org()
+        if 'user' in kwargs:
```

Comment:
django-rest-framework does this when you do a GET while accepting text/html to show the documentation.